eventkit_cloud/ui/static/ui/app/components/MapTools/SearchAOIToolbar.js

```diff
@@ -100,22 +110,35 @@ export class SearchAOIToolbar extends Component {
                         options={this.state.suggestions}
                         onChange={this.handleEnter}
                         placeholder={'Search admin boundary or location...'}
-                        onInputChange={this.debouncer}
+                        onInputChange={this.handleInputChange}
                         labelKey={'name'}
                         paginate={false}
                         emptyLabel={''}
                         minLength={2}
                         renderMenu={(results, menuProps) => {
                             return(
                                 <Menu {...menuProps}>
-                                    {results.map((result, index) => (
-                                        <TypeaheadMenuItem result={result} index={index} key={index}/>
-                                    ))
-                                    }
-                                </Menu>        
-                            )
+                                    {(this.state.fetched)
+                                    ?
+                                        (results.length)
+                                        ?
+                                            results.map((result, index) => (
+                                                <TypeaheadMenuItem result={result} index={index} key={index}/>
+                                            ))
+                                        :
+                                            <div style={{padding: '16px', userSelect: 'none', cursor: 'default'}}>No results</div>
```

Comment:
Do you think the 'No results' should be styled like an actual result? If so could you add a top border to separate the text input and the no result message? Like this borders: 
![image](https://user-images.githubusercontent.com/16799449/30060101-ee4f418c-920f-11e7-90e7-235697577716.png)
If you think it would be confusing to style it like a result we can just leave this as is.

Reply:
I totally missed those borders. I'll get those added in there for no results.